tests/tools/oesign-engine/test-enclave/CMakeLists.txt

```diff
@@ -16,6 +16,14 @@ else()
     set(SIMULATION OFF)
 endif()
 
+#Generate an empty file
+add_custom_command(OUTPUT empty.conf
+	COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/oesign-test.py 0)
```

Comment:
Do you mean using the newly created script instead of the original one?

Reply:
no, no , this script is just to generate test required config files. We are still using the original script to run test and provide result.

Reply:
Then, can you point me where is the newly added script being used?

Reply:
the script is to generate empty.conf and neg.conf, 2 config files for oesign tool test(type of negative test)

Reply:
There my be some confusion.

My original question is that should the command be
```	
        add_custom_command(OUTPUT empty.conf
	COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/oesign-custmize-cfg.py 0)
```

Instead of

```
       add_custom_command(OUTPUT empty.conf
	      COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/oesign-test.py 0)
```

Otherwise, I didn't see the `oesign-custmize-cfg.py` is used anywhere in this PR.

Reply:
ok, yes, I forgot to add the dir before this script. I will add it. thank you so much.

Reply:
you're welcome!